doc/notebooks/Dask.ipynb

```diff
@@ -445,17 +445,17 @@
        "    title:        4x daily NMC reanalysis (1948)\n",
```

Comment:
<p>Maybe add a comment on the warnings, so no one gets spooked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/xESMF/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>